tests/geometry/functional/test_so3.py

```diff
@@ -37,4 +41,33 @@ def test_inverse(batch_size: int, dtype: torch.dtype):
     rng.manual_seed(0)
     tangent_vector = torch.rand(batch_size, 3, dtype=dtype, generator=rng)
     group = so3.exp(tangent_vector)
+
+    # check analytic backward for the operator
     check_lie_group_function(so3, "inverse", TEST_EPS, group)
```

Comment:
This is again all duplicated code, can we parameterize these as suggested in the other PR?

Reply:
I'm planning to add additional tests for each function, not only about analytical backward.

Reply:
Should be okay to proceed with reviewing and merging the stream of current PRs, but let's remember to do a pass to resolve any code duplication once all the initial implementation converges.